sql/geometry.go

```diff
@@ -61,6 +61,10 @@ const (
 
 	PointSize = 16
 	CountSize = 4
+
+	PointTypeID      = 1
+	LineStringTypeID = 2
+	PolygonTypeID    = 3
```

Comment:
These identifiers should already be available below (`WKBPointID`, `WKBLineID`, etc, starting on line 73), so you can remove these.